### PR TITLE
🌐: Update simplified Chinese translation

### PR DESCRIPTION
**Pull request by snekkenull**
# Pull Request Template

## Summary

Supplement the missing Simplified Chinese translation, optimize the expression, and adjust the order to make it correspond to the English, so as to facilitate the subsequent updates.

## Change Type

Please delete any irrelevant options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation update 

## Testing

Please describe your test process and include instructions so that we can reproduce your test. If there are any important variables for your testing configuration, list them here.

## Checklist

- [ ] My code adheres to this project's style guidelines
- [ ] I have performed a self-review of my own code
- [ ] I have commented in any complex areas of my code
- [ ] I have made pertinent documentation changes
- [ ] My changes do not introduce new warnings
- [ ] I have written tests demonstrating that my changes are effective or that my feature works
- [ ] Local unit tests pass with my changes
- [ ] Any changes dependent on mine have been merged and published in downstream modules.

### CODE REVIEW

File: client/src/localization/languages/Zh.tsx [hunk 1]
```diff
@@ -187,26 +247,34 @@ export default {
...
+  com_nav_lang_french: 'Français ',
+  com_nav_lang_italian: 'Italiano',
+  com_nav_lang_polish: 'Polski',
+  com_nav_lang_brazilian_portuguese: 'Português Brasileiro',
+  com_nav_lang_russian: 'Русский',
+  com_nav_lang_japanese: '日本語',
+  com_nav_lang_swedish: 'Svenska',
+  com_nav_lang_korean: '한국어',
+  com_nav_lang_vietnamese: 'Tiếng Việt',
+  com_nav_lang_traditionalchinese: '繁體中文',
```
Comment (berry-13) [inline client/src/localization/languages/Zh.tsx]: Can you remove these translations, please?

**Review by berry-13 — CHANGES_REQUESTED** [review]

File: client/src/localization/languages/Zh.tsx (hunk 1 above)
Comment (snekkenull) [reply client/src/localization/languages/Zh.tsx]: OK

**Review by snekkenull — COMMENTED** [review]
Update

**Review by berry-13 — APPROVED** [review]
Thanks!

File: client/src/localization/languages/Zh.tsx [hunk 2]
```diff
@@ -120,64 +144,100 @@ export default {
...
-  com_endpoint_config_key_google_vertex_api: '在Google云上(Google Cloud)，然后',
-  com_endpoint_config_key_google_service_account: '创建Service Account',
-  com_endpoint_config_key_google_vertex_api_role: '确保单击“创建并继续”以至少授予“Vertex AI 用户”角色。 最后，创建一个要在此处导入的 JSON 密钥。',
-  com_endpoint: '端口',
+  com_endpoint_new_topic: '新主题',
+  com_endpoint: '渠道',
   com_endpoint_hide: '隐藏',
   com_endpoint_show: '显示',
-  com_endpoint_examples: ' 例子',
+  com_endpoint_examples: ' 预设',
```
Comment (NoahDragon) [inline client/src/localization/languages/Zh.tsx:184]: Could you please elaborate more the reason why translate the `examples` to `presets`?

File: client/src/localization/languages/Zh.tsx [hunk 3]
```diff
@@ -74,19 +94,23 @@ export default {
...
   com_auth_click_here: '点击这里',
-  com_auth_to_try_again: '再试一次.',
+  com_auth_to_try_again: '再试一次。',
   com_auth_submit_registration: '注册提交',
   com_auth_welcome_back: '欢迎',
   com_endpoint_open_menu: '打开菜单',
   com_endpoint_bing_enable_sydney: '启用 Sydney',
   com_endpoint_bing_to_enable_sydney: '启用 Sydney',
-  com_endpoint_bing_jailbreak: '破解',
+  com_endpoint_bing_jailbreak: '越狱',
```
Comment (NoahDragon) [inline client/src/localization/languages/Zh.tsx:113]: I understand the jailbreak literaly means the `越狱`, but `cracked` would be more suitable to the context and easy understand for non IT people.

File: client/src/localization/languages/Zh.tsx [hunk 4]
```diff
@@ -7,18 +7,17 @@ export default {
   com_ui_example_10_year_old_b_day: '如何举办生日宴才能耳目一新？',
   com_ui_example_http_in_js: '如何在Python中实现HTTP请求？',
   com_ui_capabilities: '功能',
-  com_ui_capability_remember: '历史对话记录',
-  com_ui_capability_correction: '实时提供反馈',
-  com_ui_capability_decline_requests: '阻止非法请求',
+  com_ui_capability_remember: '记忆历史对话',
```
Comment (NoahDragon) [inline client/src/localization/languages/Zh.tsx:10]: Feel `保存历史对话` would be more fit here.

File: client/src/localization/languages/Zh.tsx (hunk 2 above)
Comment (NoahDragon) [reply client/src/localization/languages/Zh.tsx:184]: nvm, the variable name is a bit confusing here, the actual content is `preset`.

**Comment by snekkenull** [comment]
谢谢提问。
“越狱”在该场景中是更为普遍和准确的说法，得益于自媒体传播，普通用户容易获得越狱相关的教程，例如用户在中文网络搜索“GPT越狱”获得的信息准确度会高于"GPT破解"。此外在日常使用中，“破解”可能更多意味着“把收费变免费的手段”。

同样地，“记忆历史内容”也是，原文体现的是上下文历史信息的记忆能力（在请求中附带的历史信息）而非保存能力，保存更倾向于为信息存储功能。

希望以上能够传达我的想法，欢迎交流。

Thanks for the question.
"Jailbreak" is a more common and accurate term in this place, thanks to the
spread of social media, it is easy for ordinary users to obtain
jailbreak-related tutorials, for example, users searching for "GPT
jailbreak" in the Chinese network will get more accurate information than
"GPT crack". For example, users searching for "GPT jailbreak" in Chinese
networks will get more accurate information than "GPT crack". In addition,
in daily use, "crack" may mean "a means of turning a charge into a freebie".

Similarly, "memorize historical content" is also a case where the original
text reflects the ability to memorize contextual historical information
(historical information attached to a request) rather than the ability to
preserve it, which is more of an information storage function.

I hope the above conveys my thoughts,
Let's share and improve together.


Abner Chou ***@***.***> 于 2023年11月10日周五 01:46写道：

**Comment by danny-avila** [comment]
@NoahDragon thank you for your input, and @snekkenull for your contribution. If there is an agreement, I will merge

**Comment by NoahDragon** [comment]
@danny-avila Sorry, that I missed the previous comments. But megering it is okay, these are nuances and shouldn't affect user experience. 

@snekkenull Thanks for the context. Agree that the “越狱” is more common in this use case, although I feel their differences are not distiguishable ([ref](https://www.kaspersky.com/resource-center/definitions/what-is-jailbreaking)).

For the other translation about `com_ui_capability_remember`, sorry for my oversight, you are absolutely correct.

**Pull request merged**